src/test/App.test.tsx

```diff
@@ -273,6 +274,49 @@ describe("App", () => {
     );
   });
 
+  // it.only("swipe transitions correctly on music mode", async () => {
+
+  //   await page.setViewport({ width: 2000, height: 1500 });
+  //   if (hasMultipleBooks) {
+  //     await page.waitForSelector(selectors.shlSongbook);
+  //     await page.click(selectors.shlSongbook);
+  //   }
+
+  //   await page.waitForSelector(selectors.searchViewIonCardTitle);
+
+  //   const ionCards = await page.$$(selectors.searchViewIonCardTitle);
+  //   await ionCards[5].click();
+
+  //   await page.waitForSelector(selectors.lyricViewIonCardTitle);
+
+  //   expect(page.url()).toEqual(getSongLink(6));
+  //   expect(await page.$eval(selectors.lyricViewIonCardTitle, (e) => e.innerHTML)).toEqual("Come, Thou Almighty King");
+
+
+  //   await page.waitForSelector(selectors.prevButton);
+  //   await page.evaluate(() => {
+  //     document.onmousemove = function(e: MouseEvent){
+  //       const mouseX = e.offsetX;
+  //       const mouseY = e.offsetY;
+  //       console.log('MOVEDDDDD: ', mouseX, mouseY);
+  //     }
+  //  })
+  //   await page.mouse.move(100, 221);
+  //   await page.mouse.down();
+  //   await page.mouse.move(100, 221);
+  //   await page.mouse.up();
+  //   // doesn't move
+  //   expect(page.url()).toEqual(getSongLink(6));
+
+  //   await page.mouse.move(100, 200);
+  //   await page.mouse.down();
+  //   await page.mouse.move(200, 200);
+  //   await page.mouse.up();
+
+  //   expect(page.url()).toEqual(getSongLink(6));
```

Comment:
welp, I tried to write a test but it just won't work. I think I spent more time trying to the test to work than I did working on the feature. the mouse up and mouse move is working, I can see the position changing by logging it out. But the simulated swiping just isn't working. No matter how I simulate the swipe, it always navigates to the previous song/hymn.

Reply:
just tried to upgrade puppeteer and use the new mouse drag function, it's exceeding time limit of the test even when I put 50 sec on the test. I'm kinda out of ideas, maybe we could just simulate mouse dragging with pure js for the test.

![image](https://user-images.githubusercontent.com/76019114/167918510-4525c2eb-a277-4523-8705-240f6f508b6b.png)

Reply:
thanks dad, I tested out `drag()` but it was timing out. I'll look into the other drag function

Reply:
can remove i think